onnxruntime/core/mlas/inc/mlas_q4.h

```diff
@@ -61,18 +68,9 @@
  * @param N          the number of columns of matrix B.
  * @param K          the number of rows of matrix B.
  * @param ldb        leading dimension of B
-*/
-void
-MLASCALL
-MlasQ4GemmPackB(
-    MLAS_BLK_QUANT_TYPE QType,
-    void* PackedBuf,
-    const float* FpData,
-    size_t N,
-    size_t K,
-    size_t ldb
-    );
-
+ */
+void MLASCALL
+MlasQ4GemmPackB(MLAS_BLK_QUANT_TYPE QType, void* PackedBuf, const float* FpData, size_t N, size_t K, size_t ldb);
```

Comment:
could you please leave the format as it is?

Reply:
ok, NP!  Does mlas have a different format file? I ran clang-format -style=file, and all these formats changed.

Reply:
all format codes are reverted.